### PR TITLE
activex-scripting -- Fixes for Microsoft Scripting Runtime definitions

### PR DESCRIPTION
**Pull request by zspitz**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/activex-excel/index.d.ts [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.6
 
 /// <reference types="activex-office" />
 /// <reference types="activex-vbide" />
 /// <reference types="activex-stdole" />
 
 declare namespace Excel {
+    // tslint:disable-next-line:no-const-enum
```
Comment (DanielRosenwasser) [inline types/activex-excel/index.d.ts]: I think that the most appropriate change would be to change these to regular enums.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/activex-excel/index.d.ts (hunk 1 above)
Comment (zspitz) [reply types/activex-excel/index.d.ts]: @DanielRosenwasser These enums don't exist at runtime; in other words the following Javascript will fail:

    var x = Excel.Constants.xl3DBar;

But they are still useful as [documented constant values](https://github.com/Microsoft/TypeScript/issues/20703#issuecomment-361914358).

I understand this is a problem when `--isolatedModules`, but my understanding was that the final fix will be to [error on files consuming enums this way when `--isolatedModules` has been set.](https://github.com/Microsoft/TypeScript/issues/20703#issuecomment-361434795).

Comment (zspitz) [reply types/activex-excel/index.d.ts]: @DanielRosenwasser Any further word on this?

**Comment by DanielRosenwasser** [comment]
You *probably* shouldn't need all of those suppression comments, but if that's a problem with CI, this is fine.

**Comment by zspitz** [comment]
@DanielRosenwasser Would a setting in `tslint.json` be preferable?

**Comment by DanielRosenwasser** [comment]
Yeah, overall I'd say so.

**Pull request merged**